src/dataset-config.js

```diff
@@ -0,0 +1,518 @@
+'use strict';
+
+const DATASET_ATTRIBUTIONS = {
```

Comment:
Why do we need to store these? The `/data/availability` endpoint returns the attributions of the dataset.